webapp/Connector/src/view/Learn.js

```diff
@@ -726,6 +734,7 @@ Ext.define('Connector.view.LearnHeader', {
             this.searchField = Ext.create('Ext.form.field.Text', {
                 emptyText: 'Search',
                 cls: 'learn-search-input',
+                width: 300,
```

Comment:
Related to previous comment about missing flex config, would using flex and minWidth instead provide better user experience or is a fixed width of 300px the desired behavior?